compiler/luci/pass/src/QuantizeWithMinMaxPass.cpp

```diff
@@ -687,6 +687,31 @@ struct QuantizeBias final : public luci::CircleNodeMutableVisitor<void>
   loco::DataType output_type;
   QuantizationGranularity granularity;
 
+private:
+  struct IWB
+  {
+    CircleNode *input = nullptr;
+    CircleNode *weights = nullptr;
+    CircleConst *bias = nullptr;
+
+    IWB(loco::Node *i, loco::Node *w, loco::Node *b)
+    {
+      input = dynamic_cast<luci::CircleNode *>(i);
+      weights = dynamic_cast<luci::CircleNode *>(w);
```

Comment:
AFAIK, we have a convention that does not throw an exception in constructor. So I used dynamic_cast instead of loco::must.